scipy/stats/stats.py

```diff
@@ -8121,6 +8121,237 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
     return BrunnerMunzelResult(wbfn, p)
 
 
+def _confint_lowerbound(n, quantile, confidence):
+    r"""
+    Compute the lower bound for a one-sided confidence interval
```

Comment:
Looks like I messed this up. Please add all the whitespace back. 

Maybe the fastest way would be to copy-paste your code from before the merge [here](https://github.com/scipy/scipy/blob/790cd76100e26aa3b4ac82bd70bce631d14be48f/scipy/stats/stats.py).